CMakeLists.txt

```diff
@@ -186,6 +186,7 @@ target_link_directories(${PROJECT_NAME} PRIVATE
 
 add_custom_command(TARGET ${PROJECT_NAME} POST_BUILD 
 											COMMAND size -A ${PROJECT_NAME}
+											COMMAND mkdir -p ${PROJECT_SOURCE_DIR}/out
 											COMMAND cp ${PROJECT_NAME} ${PROJECT_SOURCE_DIR}/out
 											COMMAND cp ${PROJECT_NAME} ${OS_APP_PATH}
 											COMMAND cd ${OS_APP_PATH} && cd ../../ && rm -f ./ota_app/hdzgoggle_app_ota*.tar && ./mkapp_ota && ./make_fw.sh
```

Comment:
Could it be a good idea to change `cd ${OS_APP_PATH} && cd ../../` to `cd ${PROJECT_SOURCE_DIR}/mkapp/`

Reply:
...and add `/` in end of the move command below? `${PROJECT_SOURCE_DIR}/out` -> `${PROJECT_SOURCE_DIR}/out/`

Reply:
Changes made as suggested 👍